pilot/pkg/kube/inject/testdata/webhook/daemonset.yaml.injected

```diff
@@ -110,14 +112,15 @@ spec:
         - "80"
         - -d
         - "15020"
-        image: docker.io/istio/proxy_init:unittest
+        image: gcr.io/istio-release/proxy_init:master-latest-daily
         imagePullPolicy: IfNotPresent
         name: istio-init
         resources: {}
         securityContext:
           capabilities:
             add:
             - NET_ADMIN
+          privileged: true
```

Comment:
What does this mean?

Reply:
I am not 100% sure. That seems to be the default value and the Marshal now writes it to the file. @andraxylia Can you comment what does this "privileged" mean?